### PR TITLE
Update fc.rosinstall for version 2.0.0

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: fc.rosinstall [hunk 1]
```diff
@@ -1,60 +1,32 @@
...
-    uri: https://github.com/code-iai/iai_kinect2.git
+    local-name: FOR_LATEST_BAXTEREUS/baxter_interface
+    uri: https://github.com/wkentaro/baxter_interface.git
+    version: baxtereus-devel
+- git:
+    local-name: FOR_LATEST_BAXTEREUS/jsk_robot
+    uri: https://github.com/jsk-ros-pkg/jsk_robot.git
     version: master
+- git:
+    local-name: FOR_PCL1.8/octomap_mapping
```
Comment (knorth55) [inline fc.rosinstall]: `FOR_PCL1.8/OctoMap/octomapping` is better, isnt it?

File: fc.rosinstall [hunk 2]
```diff
@@ -1,60 +1,32 @@
...
+    uri: https://github.com/OctoMap/octomap_mapping.git
+    version: indigo-devel
+- git:
+    local-name: FOR_PCL1.8/perception_pcl
+    uri: https://github.com/ros-perception/perception_pcl.git
```
Comment (knorth55) [inline fc.rosinstall]: same as above

File: fc.rosinstall (hunk 1 above)
Comment (wkentaro) [inline fc.rosinstall]: I'm not sure which should I use if
- https://github.com/wkentaro/baxter_interface.git
- RethinkRobotics/baxter_interface or wkentaro/baxter_interface

That's why I chosen this, and too long local-name breaks readability of wstool info command output.
So I decided to use current local-name. Any idea?

Comment (knorth55) [inline fc.rosinstall]: no idea.
it seems that your opinion is also correct.
anyway, i follow your idea.

**Pull request merged**